R/system.R

```diff
@@ -356,6 +359,15 @@ getDBConnection <- function(type, host, port, databaseName, username, password,
     loadNamespace("RPresto")
     drv <- RPresto::Presto()
     conn <- RPresto::dbConnect(drv, user = username, password = password, host = host, port = port, schema = schema, catalog = catalog, session.timezone = Sys.timezone(location = TRUE))
+  } else if (type == "odbc") {
+    loadNamespace("RODBC")
+    connstr <- stringr::str_c("RODBC::odbcConnect(dsn = '",dsn, "',uid = '", username, "', pwd = '", password, "'")
```

Comment:
Could you add space between comma and dsn for unified coding style? 
`,dsn` -> `, dsn`